docs: Update PNPM commands

ThanatosGit opened the pull request:
## Information

| Type                  | Breaking change |
| --------------------- | --------------- |
| Doc| No          |

---

Installation command from https://tsed.dev/introduction/getting-started.html is wrong.

```
thanatos@DESKTOP-5NP8L7E:~/projects/tsed-repos$ pnpm --version
9.15.4

thanatos@DESKTOP-5NP8L7E:~/projects/tsed-repos/jest-test$ pnpx -p @tsed/cli tsed init .
/home/thanatos/.cache/pnpm/dlx/2eaiiergvzc3777g5qgjrb7sim/19474c44e0b-564:
 ERR_PNPM_FETCH_404  GET https://registry.npmjs.org/-p: Not Found - 404

This error happened while installing a direct dependency of /home/thanatos/.cache/pnpm/dlx/2eaiiergvzc3777g5qgjrb7sim/19474c44e0b-564

-p is not in the npm registry, or you have no permission to fetch it.

No authorization header was set for the request.
```
### Fix

Instead of using the outdated/deprecated `pnpx` use `pnpm dlx` and also add peer dependencies.

E.g.
```bash
pnpm --package=@tsed/cli --package=@tsed/cli-core dlx tsed init .
```


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Documentation**
  - Updated project initialization commands in multiple documentation files to use the latest `pnpm` syntax.
  - Replaced `pnpx -p @tsed/cli tsed init` with `pnpm --package=@tsed/cli dlx tsed init .` and `pnpx -p @tsed/cli tsed init tsed-prisma` with `pnpm --package=@tsed/cli dlx tsed init tsed-prisma`.
  - Affected files: getting-started.md, migrate-from-express.md, prisma.md.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

ThanatosGit committed:
docs: Update PNPM commands

Romakita commented:
Hi @ThanatosGit 
Thanks for the. Why pnpm command needs to have peerdeps declared explicitly ? It complexify the command line for nothing because @tsed/cli-core is dependency of @tsed/cli.

See you

Romakita commented:
Arf... the `@tsed/cli-core` is a peer not a dependencies.

```json
  "peerDependencies": {
    "@tsed/cli-core": "workspace:*",
    "@tsed/core": ">=8.0.0",
    "@tsed/di": ">=8.0.0",
    "@tsed/hooks": ">=8.0.0",
    "@tsed/logger": ">=7.0.1",
    "@tsed/openspec": ">=8.0.0",
    "@tsed/schema": ">=8.0.0"
  }
```
But it should be the same issue with other peerDependencies listed above.

Romakita commented:
@ThanatosGit can you retrieve install using `pnpm --package=@tsed/cli dlx tsed init .`, please ?

ThanatosGit commented:
```
thanatos@DESKTOP-5NP8L7E:~/projects/tsed-repos/init-test$ pnpm --package=@tsed/cli dlx tsed init .
 WARN  1 deprecated subdependencies found: read-pkg-up@11.0.0
Packages: +83
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 105, reused 87, downloaded 1, added 83, done
 WARN  Issues with peer dependencies found
.
└─┬ @tsed/cli 6.1.7
  └── ✕ missing peer @tsed/cli-core@"*"
Peer dependencies that should be installed:
  @tsed/cli-core@"*"  
node:internal/modules/esm/resolve:857
  throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath(base), null);
        ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@tsed/cli-core' imported from /home/thanatos/.cache/pnpm/dlx/rnzi4ehbwa6qos5yfkxu3ogi6a/19475a84609-ed6/node_modules/.pnpm/@tsed+cli@6.1.7_@swc+types@0.1.17_@tsed+core@8.4.2_@tsed+di@8.4.2_@tsed+core@8.4.2_@tsed+hook_tsq4m65iv5dbdjdvk6wwew7uhq/node_modules/@tsed/cli/lib/esm/bin/tsed.js
    at packageResolve (node:internal/modules/esm/resolve:857:9)
    at moduleResolve (node:internal/modules/esm/resolve:926:18)
    at defaultResolve (node:internal/modules/esm/resolve:1056:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:654:12)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:603:25)
    at #resolveAndMaybeBlockOnLoaderThread (node:internal/modules/esm/loader:619:38)
    at ModuleLoader.resolveSync (node:internal/modules/esm/loader:636:52)
    at Object.resolve (node:internal/modules/esm/initialize_import_meta:33:25)
    at file:///home/thanatos/.cache/pnpm/dlx/rnzi4ehbwa6qos5yfkxu3ogi6a/19475a84609-ed6/node_modules/.pnpm/@tsed+cli@6.1.7_@swc+types@0.1.17_@tsed+core@8.4.2_@tsed+di@8.4.2_@tsed+core@8.4.2_@tsed+hook_tsq4m65iv5dbdjdvk6wwew7uhq/node_modules/@tsed/cli/lib/esm/bin/tsed.js:11:53
    at ModuleJob.run (node:internal/modules/esm/module_job:271:25) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v22.13.0
```

Romakita commented:
Ok it's fixed now we can use the following command to install cli correctly:

```sh
pnpm --package=@tsed/cli@latest dlx tsed init .
```

@ThanatosGit can you update your PR please ;)

Romakita commented:
You need also to update the this file: https://github.com/tsedio/tsed-vitepress-theme/blob/main/packages/theme/organisms/home/terminal/HomeTabTerminalPnpm.vue

ThanatosGit commented:
I left out `@latest` because the other cmd's don't contain it either
https://github.com/tsedio/tsed-vitepress-theme/pull/110

Romakita commented:
I added latest for pnpm command because pnpm doesn't install the latest version if you already installed the cli using dlx. 

Npm and yarn always try to install latest.

ThanatosGit commented:
I wasn't aware of that. I tried to find some documentation about it and tried to reproduce. 

See console log at the end: That might be because I explicitly requested an older version first and then omitted the version but it was pulling the newer version (because the prompt worked at the second attempt).

According to this comment `pnpm` even caches the `latest` tag https://github.com/pnpm/pnpm/issues/8659#issuecomment-2488128352 and this might be why it was cached to an older version for you? 

Anyways: Should I include `@latest` now or leave it as it is?

```bash
thanatos@DESKTOP-5NP8L7E:~/projects/tsed-repos$ rm -rf /home/thanatos/.cache/pnpm/
thanatos@DESKTOP-5NP8L7E:~/projects/tsed-repos$ echo  $(pnpm store path)
/home/thanatos/.local/share/pnpm/store/v3
thanatos@DESKTOP-5NP8L7E:~/projects/tsed-repos$ rm -rf $(pnpm store path)
thanatos@DESKTOP-5NP8L7E:~/projects/tsed-repos$ pnpm --package=@tsed/cli@6.1.4 dlx tsed init .
Downloading @swc/core-linux-x64-gnu@1.7.26: 17.31 MB/17.31 MB, done
Downloading @swc/core-linux-x64-musl@1.7.26: 21.01 MB/21.01 MB, done
 WARN  1 deprecated subdependencies found: read-pkg-up@11.0.0
Packages: +83
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 105, reused 0, downloaded 88, added 83, done
.cache/pnpm/dlx/v2cti3auz46hvfggoeecbbntc4/194761c48a7-6eda/node_modules/.pnpm/@swc+core@1.7.26_@swc+helpers@0.5.15/node_modules/@swc/core: Running postinstall script, done in 21ms
 WARN  Issues with peer dependencies found
.
└─┬ @tsed/cli 6.1.4
  └── ✕ missing peer @tsed/cli-core@"*"
Peer dependencies that should be installed:
  @tsed/cli-core@"*"  
node:internal/modules/esm/resolve:857
  throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath(base), null);
        ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@tsed/cli-core' imported from /home/thanatos/.cache/pnpm/dlx/v2cti3auz46hvfggoeecbbntc4/194761c48a7-6eda/node_modules/.pnpm/@tsed+cli@6.1.4_@swc+types@0.1.17_@tsed+core@8.4.2_@tsed+di@8.4.2_@tsed+core@8.4.2_@tsed+hook_ubsk4icuuhyc7nlzmv3upozjh4/node_modules/@tsed/cli/lib/esm/bin/tsed.js
    at packageResolve (node:internal/modules/esm/resolve:857:9)
    at moduleResolve (node:internal/modules/esm/resolve:926:18)
    at defaultResolve (node:internal/modules/esm/resolve:1056:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:654:12)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:603:25)
    at #resolveAndMaybeBlockOnLoaderThread (node:internal/modules/esm/loader:619:38)
    at ModuleLoader.resolveSync (node:internal/modules/esm/loader:636:52)
    at Object.resolve (node:internal/modules/esm/initialize_import_meta:33:25)
    at file:///home/thanatos/.cache/pnpm/dlx/v2cti3auz46hvfggoeecbbntc4/194761c48a7-6eda/node_modules/.pnpm/@tsed+cli@6.1.4_@swc+types@0.1.17_@tsed+core@8.4.2_@tsed+di@8.4.2_@tsed+core@8.4.2_@tsed+hook_ubsk4icuuhyc7nlzmv3upozjh4/node_modules/@tsed/cli/lib/esm/bin/tsed.js:11:53
    at ModuleJob.run (node:internal/modules/esm/module_job:271:25) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v22.13.0
thanatos@DESKTOP-5NP8L7E:~/projects/tsed-repos$ pnpm --package=@tsed/cli dlx tsed init .
 WARN  1 deprecated subdependencies found: read-pkg-up@11.0.0
Packages: +359
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 381, reused 87, downloaded 277, added 359, done
.cache/pnpm/dlx/rnzi4ehbwa6qos5yfkxu3ogi6a/194761c7b13-6f91/node_modules/.pnpm/@swc+core@1.7.26_@swc+helpers@0.5.15/node_modules/@swc/core: Running postinstall script, done in 21ms
? Choose the target Framework: (Use arrow keys)
❯ Express.js 
  Koa.js 
thanatos@DESKTOP-5NP8L7E:~/projects/tsed-repos$ 
```

Romakita commented:
we will add `@latest` later if it's an issue. right know it's ok ;)

Romakita commented:
:tada: This PR is included in version 8.4.3 :tada:

The release is available on:
- `v8.4.3`
- [GitHub release](https://github.com/tsedio/tsed/releases/tag/v8.4.3)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
